### PR TITLE
Update Ram icon

### LINKED ISSUE #11306: Update: Ram
### Brand Name

Ram

### Official Resources for Icon and Color

The text under the logo is unnecessary, we'll need to remove it, so that will remain the Ram icon.

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by ahmadnadeem6236**
**Issue:** closes #11306.

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by uncenter** [comment]
Please do not ignore our pull request template.

File: icons/ram.svg
Comment (uncenter) [inline icons/ram.svg:1]: If you had followed the pull request template you'll have been asked to check a box stating whether you had optimized the icon. It is clear you have not, nor have you followed [that step in our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon).

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/ram.svg
Comment (ahmadnadeem6236) [reply icons/ram.svg:1]: Okay I will use the optimize tool to optimize the svg

**Comment by ahmadnadeem6236** [comment]
Please go through the change. I followed the pull request template

File: icons/ram.svg
Comment (uncenter) [reply icons/ram.svg:1]: The icon must be all on one line, not prettified/formatted like you have it here.

Comment (ahmadnadeem6236) [reply icons/ram.svg:1]: Done it

**Comment by uncenter** [comment]
Please see the linter errors. You can run the linter locally with `npm run lint` (make sure to run `npm install` first).

**Comment by adamrusted** [comment]
See https://github.com/simple-icons/simple-icons/pull/5014#issuecomment-779194896 - though that resource is no longer available, it still stands unless you've found guidelines that say otherwise.

**Comment by uncenter** [comment]
@ahmadnadeem6236 are you still interested in working on this pull request?

**Comment by uncenter** [comment]
Closing as no response from PR author, can reopen if you have still have an interest in working on it! Future contributors- feel free to pick up where this left off in a new PR.

**Pull request closed without merging**